README.md

```diff
@@ -336,11 +328,51 @@ Finally once all the attached events have been processed, the properties lifecyc
 
 <!-- TODO: render lifecycle goes here -->
 
-##### AfterRender
+Occassionally, in a mixin or base widget class, it my be required to provide logic that needs to be executed before or after a widgets' `render` call. These lifecycle hooks are supported in `WidgetBase` and operate as before and after aspects. 
```

Comment:
widgets' -> widget's (unless you meant widgets to be both plural and possessive)